### DISCUSSION TITLE
About the release cycle

### DISCUSSION THREAD

**Discussion by yujeong-jeon** [NaverPayDev/hidash#128] [Q&A] [answered]
Before leaving 1.0.0, I would like to inquire about the distribution cycle of the development version.
Should I look at it as collecting it once a week and going out?

1.0.0 나가기 전, 개발 버전의 배포 주기를 문의드립니다.
일주일에 한 번씩 모아서 나가는 것으로 보면 될까요?

**Comment by yceffort-naver** [comment, marked as answer]
🤔 0.x 버전이라 그냥 아무 때나 무지성으로 내보내려고 헀습니다 ㅋㅋ (별 생각 없었..)
의견 있으시면 말씀 부탁드립니다

**Reply by yujeong-jeon** [reply]
아하 혹시 매주 금요일에 모아다가 나가는 건 어떨까요~??
버전 업데이트가 귀찮다면 주기적으로 레지스트리에 hidash 버전 읽어서 업데이트해주는 로직을 GIthub App으로 지원하도록 하겠습니다 :) (회사 한정 ㅋㅋ)

**Reply by yceffort-naver** [reply]
좋습니다!!!!

**Reply by yujeong-jeon** [reply]
### Translations

> It's a 0.x version, so I just tried to send it out with ignorance at any time lol (I didn't think much about it..)
If you have any ideas, please let me know

> Aha, how about collecting them every Friday and releasing??
If it's annoying to update the version, I'll support the Github app with the logic of updating the hidash version in the registry periodically :) (Company only lol)

> I think it's good!!
